### DISCUSSION TITLE
<localleader>E doesn't work

### DISCUSSION THREAD

**Discussion by lispercat** [Olical/conjure#570] [General]
I guess it's a question of settings but when I am in visual mode and select a line, pressing my local leader "\" doesn't trigger and command, looks like it's disabled. Is there a setting I can enabled it with for conjure?

**Comment by Olical** [comment]
So all I can say right now is that it works on my machine, which doesn't help at all. Would you be able to share any custom Conjure configuration you have? Maybe you have a custom mapping defined in visual mode (or you have a plugin that does) that overrides Conjure's mapping?
